browser/playlist/test/playlist_download_request_manager_browsertests.cc

```diff
@@ -51,32 +97,50 @@ class PlaylistDownloadRequestManagerBrowserTest : public PlatformBrowserTest {
     return item;
   }
 
+  GURL LoadHTML(const std::string& html, GURL url = GURL()) {
+    const auto* test_info =
+        testing::UnitTest::GetInstance()->current_test_info();
+    VLOG(2) << __FUNCTION__ << test_info->name() << ": " << __func__;
+
+    // Start server: ASSER_TRUE() has its own return type internally, thus
```

Comment:
nit:
```suggestion
    // Start server: ASSERT_TRUE() has its own return type internally, thus
```

Reply:
Cool! I didn't think of doing this! That's a cool way of being able to `ASSERT` anywhere!